client.go

```diff
@@ -608,12 +652,67 @@ func (s *Client) LoopOutQuote(ctx context.Context,
 		return nil, err
 	}
 
-	return &LoopOutQuote{
+	loopOutQuote := &LoopOutQuote{
 		SwapFee:         quote.SwapFee,
 		MinerFee:        minerFee,
 		PrepayAmount:    quote.PrepayAmount,
 		SwapPaymentDest: quote.SwapPaymentDest,
-	}, nil
+	}
+
+	// If we use an Asset we'll rfq to get the asset amounts to use for
+	// the swap.
+	if request.AssetRFQRequest != nil {
+		rfqReq := request.AssetRFQRequest
+		if rfqReq.Expiry == 0 {
```

Comment:
Should we check that expiry and limit multiplier are sane? As in they're non negative etc.